DotNetHowTo/DotNetHowTo/Address.Methods.cs

```diff
@@ -20,14 +21,11 @@ public override string ToString()
             return builder.ToString();
         }
 
-        public bool IsEmpty {
-            get {
-                return (String.IsNullOrEmpty(StreetAddress) &&
-                        String.IsNullOrEmpty(City) &&
-                        String.IsNullOrEmpty(StateProvince) &&
-                        String.IsNullOrEmpty(PostalCode) &&
-                        String.IsNullOrEmpty(Country));
-            }
-        }
+        [JsonIgnore]
+        public bool IsEmpty => String.IsNullOrEmpty(StreetAddress) &&
```

Comment:
nit: What were to happen to the indentation if you renamed `IsEmpty` to a longer or shorter name?

Reply:
If I changed the name, I would fix the indentation.

Reply:
If you indented differently, you wouldn't have to. :)